web/forms.py

```
@@ -527,6 +527,50 @@
+        widget=TailwindTextarea(attrs={'rows': 5, 'placeholder': 'Starter code for coding questions'}),
+        required=False,
+        help_text="Starter code for coding questions"
+    )
+    
+    expected_output = forms.CharField(
+        widget=TailwindTextarea(attrs={'rows': 3, 'placeholder': 'Expected output for coding questions'}),
+        required=False,
+        help_text="Expected output for coding questions"
+    )
+    
+    def __init__(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)